keps/sig-node/4210-max-image-gc-age/README.md

```diff
@@ -208,12 +208,13 @@ Additional tests will be added to this file to cover the garbage collection e2e.
 
 - Configuration field added to the Kubelet (disabled by default)
 - Feature supported by Kubelet Image Manager
-- Unit tests and e2e tests added
+- Unit tests
 - Add a metric `kubelet_image_garbage_collected_total` which tracks the number of images the kubelet is GC'ing through any mechanism.
 
 #### Beta
 
-- Gather feedback from users
+- Add e2e tests
+- Document `kubelet_image_garbage_collected_total` (a step missed in alpha)
```

Comment:
For PRR, I do not think this metric is sufficient as-is to detect when this feature is in-use because it increments on any image GC event.  I think adding a `reason` label or similar to distinguish between the `time` vs `space` usages will be needed here.

Reply:
updated with a beta goal below

Reply:
updated, thank you!